### PR TITLE
[shopsys] Luigi's Box brand feed

### PR DESCRIPTION
**Pull request by TomasLudvik**
| Q             | A
| ------------- | ---
|Description, the reason for the PR| Last missing feed for implementation of Luigi's Box
|New feature| Yes 
|[BC breaks](https://docs.shopsys.com/en/latest/contributing/backward-compatibility-promise/)| No 
|Fixes issues| ... 
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

----
:globe_with_meridians: Live Preview:
  - https://tl-luigis-box-brand-feed.odin.shopsys.cloud
  - https://cz.tl-luigis-box-brand-feed.odin.shopsys.cloud

### CODE REVIEW

File: docs/introduction/monorepo.md [hunk 1]
```diff
@@ -31,6 +31,7 @@ If you are interested, you can read more about the monorepo approach here - http
 `packages/`
 
 -   [shopsys/framework](https://github.com/shopsys/framework)
+-   [shopsys/brand-feed-luigis-box](https://github.com/shopsys/category-feed-luigis-box)
```
Comment (grossmannmartin) [inline docs/introduction/monorepo.md]:
```suggestion
-   [shopsys/brand-feed-luigis-box](https://github.com/shopsys/brand-feed-luigis-box)
```

File: packages/brand-feed-luigis-box/.github/workflows/run-checks-tests.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        runs-on: ubuntu-22.04
+        steps:
+            -   name: GIT checkout branch - ${{ github.ref }}
+                uses: actions/checkout@v4
+                with:
+                    ref: ${{ github.ref }}
+            -   name: Install PHP, extensions and tools
+                uses: shivammathur/setup-php@v2
+                with:
+                    php-version: '8.1'
```
Comment (grossmannmartin) [inline packages/brand-feed-luigis-box/.github/workflows/run-checks-tests.yaml]:
```suggestion
                    php-version: '8.3'
```

File: packages/brand-feed-luigis-box/composer.json [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+            "Shopsys\\BrandFeed\\LuigisBoxBundle\\": "src/"
+        }
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "Tests\\BrandFeed\\LuigisBoxBundle\\": "tests/"
+        }
+    },
+    "require": {
+        "php": "^8.1",
```
Comment (grossmannmartin) [inline packages/brand-feed-luigis-box/composer.json]:
```suggestion
        "php": "^8.3",
```

File: packages/brand-feed-luigis-box/composer.json [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+        "symfony/config": "^5.4",
+        "symfony/dependency-injection": "^5.4",
+        "symfony/form": "^5.4",
+        "symfony/http-kernel": "^5.4",
+        "symfony/translation": "^5.4"
+    },
+    "require-dev": {
+        "phpunit/phpunit": "^9.5.20",
+        "shopsys/coding-standards": "14.0.x-dev",
+        "twig/twig": "^3.5.0"
```
Comment (grossmannmartin) [inline packages/brand-feed-luigis-box/composer.json]: twig is a crucial component for this package, so it should be directly in require

File: packages/brand-feed-luigis-box/ecs.php [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+    $ecsConfig->skip([
+        ObjectIsCreatedByFactorySniff::class => [
+            __DIR__ . '/tests/*',
+        ],
+        PhpdocToPropertyTypeFixer::class => [
+            __DIR__ . '/src/*',
+        ],
+        DeclareStrictTypesFixer::class => [
+            __DIR__ . '/src/*',
+        ],
```
Comment (grossmannmartin) [inline packages/brand-feed-luigis-box/ecs.php]: Do we really want to skip the declare strict types rule for the new package?

File: packages/brand-feed-luigis-box/src/Resources/views/feed.xml.twig [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+<?xml version="1.0" encoding="utf-8"?>
+<brands>
+{% endblock %}
+
+{% block item %}
+    <brand>
+        <identity>{{ item.identity }}</identity>
+        <name>{{ item.name }}</name>
+        <url>{{ item.url }}</url>
+        <image_url>{{ item.imageUrl }}</image_url>
```
Comment (grossmannmartin) [inline packages/brand-feed-luigis-box/src/Resources/views/feed.xml.twig]: when imageUrl is null, is it ok to render empty tag?

**Review by grossmannmartin — CHANGES_REQUESTED** [review]

File: docs/introduction/monorepo.md (hunk 1 above)
Comment (TomasLudvik) [reply docs/introduction/monorepo.md]: fixed

File: packages/brand-feed-luigis-box/.github/workflows/run-checks-tests.yaml (hunk 2 above)
Comment (TomasLudvik) [reply packages/brand-feed-luigis-box/.github/workflows/run-checks-tests.yaml]: changed

File: packages/brand-feed-luigis-box/composer.json (hunk 3 above)
Comment (TomasLudvik) [reply packages/brand-feed-luigis-box/composer.json]: changed

File: packages/brand-feed-luigis-box/composer.json (hunk 4 above)
Comment (TomasLudvik) [reply packages/brand-feed-luigis-box/composer.json]: fixed

File: packages/brand-feed-luigis-box/ecs.php (hunk 5 above)
Comment (TomasLudvik) [reply packages/brand-feed-luigis-box/ecs.php]: I have removed it

File: packages/brand-feed-luigis-box/src/Resources/views/feed.xml.twig (hunk 6 above)
Comment (TomasLudvik) [reply packages/brand-feed-luigis-box/src/Resources/views/feed.xml.twig]: Nice catch, thx

**Review by grossmannmartin — APPROVED** [review]

**Pull request merged**